Pipfile

```diff
@@ -0,0 +1,24 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+apertium-streamparser = "*"
+chardet = "*"
+requests = "*"
+tornado = "*"
+commentjson = "*"
+
+[dev-packages]
+coverage = "*"
+coveralls = "*"
+flake8-commas = "==2.0.0"
+flake8-quotes = "==0.14.0"
+flake8 = "==3.5.0"
+mypy = "*"
+pep8-naming = "*"
+pycodestyle = "*"
+pyflakes = "*"
+
+
```

Comment:
There seem to be extra empty lines here.

I believe `pyflakes` and `pycodestyle` are also transitive dependencies.